cms/templates/cms/home_page.html

```diff
@@ -125,8 +131,12 @@ <h4>Proctored Tests</h4>
             <div class="reasons-images-wrap">
               <img src="{% static 'images/hiw3.png' %}" alt="">
             </div>
-            <h4>Earn a Valuable Credential</h4>
-            <p>A MicroMasters credential from MITx proves that you’ve gained the knowledge and skills to give you a competitive advantage for career success.</p>
+            <h4>Earn a Valuable Credential and Boost Your Network</h4>
+            <p>A MicroMasters program credential from MITx proves that you’ve gained the knowledge
+              and skills and knowledge to stand out in your field and qualifies you as an affiliate of
```

Comment:
```suggestion
              and skills to stand out in your field and qualifies you as an affiliate of
```

Somehow we got "knowledge" twice